src/test/resources/test.xml

```diff
@@ -12,9 +12,5 @@
   <dependencies defaultconfmapping="test->*">
     <dependency name="server" rev="${omero.version}" changing="true"/>
     <dependency name="common-test" rev="${omero.version}" changing="true"/>
-    <dependency name="blitz-test" rev="${omero.version}" changing="true"/>
```

Comment:
:+1: It's unclear to me how this ever worked as this is a dependency inversion

Reply:
It is unclear to me too.

Reply:
I would have to dig up the specifics, but the test dependency graph definitely differs from the jar dependency graph, or at least did in the mono repo.

Reply:
logically it also does not make sense to have the dependency on blitz-test since omero-server is built before.